pages/services/kafka/2.6.0-2.2.1/notes/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Release notes for 2.6.0-2.2.1
+title: Release Notes
+menuWeight: 120
+model: /services/kafka/data.yml
+render: mustache
+---
+
+## Version 2.6.0-2.2.1
+
+### Updates
+- Upgrade the base dcos-commons SDK version to `0.56.1`.
+- Upgrade {{ model.techShortName }} base tech to version `2.2.1`. See [{{ model.techShortName }}'s Release Notes](https://www.apache.org/dist/kafka/2.2.1/RELEASE_NOTES.html) for details.
```

Comment:
Let's move this up as the upgrade to 2.2.1 is more significant and comes with a big set of docs from apache

Reply:
Done